jqassistant.properties

```diff
@@ -1,15 +1,21 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.9.0,1.10.0,1.11.0
-publicVersions=1.12.0
+archivedVersions=1.9.0,1.10.0,1.11.0,1.12.0
+publicVersions=1.12.1
 
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+1.12.1.description=Improved sensor performance and ensure that all issues that cannot be attached to a file are displayed on project level
```

Comment:
I'd like to see this project-level issue in action. Can you provide me with a project to test the functionality with, please?

Reply:
I got a bunch of compile errors & no classes...?

Reply:
Sorry, my fault. It's fixed now (note to myself: never give a demo project you're currently working on to externals as reference...).

Reply:
`[ERROR] Failed to execute goal com.buschmais.jqassistant:jqassistant-maven-plugin:2.0.0-M4:scan (default-cli) on project hexagonal: Execution default-cli of goal com.buschmais.jqassistant:jqassistant-maven-plugin:2.0.0-M4:scan failed: An API incompatibility was encountered while executing com.buschmais.jqassistant:jqassistant-maven-plugin:2.0.0-M4:scan: java.lang.LinkageError: Cannot to link java.nio.DirectByteBuffer`

Reply:
Which Java version are you using?

Reply:
Extended the project config to run with JDK 17, can you pull and try again?

Reply:
So now I'm getting the "right" error?
```
[ERROR] --[ Constraint Violation ]-----------------------------------------
[ERROR] Constraint: jmolecules-ddd:IllegalDependenciesBetweenBoundedContexts
[ERROR] Severity: MAJOR
[ERROR] Number of rows: 8
[ERROR] Checks that dependencies between :BoundedContext nodes exist only where allowed.
```

Reply:
Exactly, those are expected and should be published to SonarQube. Some of them on file, some of them on project level.